### PR TITLE
Add missing headers create schedule & add upstash-callback-forward

### PR DESCRIPTION
**Pull request by CahidArda**
This pr does several things:
- ef2268166fb3d2099b6a783a1def7b8db102a499: request examples in create schedule didn't have destinations. Added example destination.
- 57f9564b4b48b984dc998cb03104301831511fab: documented upstash-callback-forward header
- 4ba54e8bf9d2dbfa5bf7854c8c24c28752fb3f57: destination field was missing in the path of create schedule. This interactive playground didn't work. It still doesn't work. More info below.
- 3cd2e35b9cb1c997e7654c6ba703bb2ee54b5c58: added upstash-schedule-id docs

### Issue in Create Schedule Playground

task for tracking https://linear.app/upstash/issue/DX-1270/docs-create-schedule-doesnt-work

Previously, the destination was missing in the path altogether:

image

this results in an error as seen above. The request doesn't even have the provided destination.

After the update, the destination exists in the request but it is escaped, which still doesn't work on QStash:

image
image

It doesn't look like the ParamField has fields which can help us in this case.
https://mintlify.com/docs/api-playground/overview
https://mintlify.com/docs/content/components/params

### CODE REVIEW

File: qstash/api/schedules/create.mdx [hunk 1]
```diff
@@ -54,12 +64,12 @@ example: "50s" | "3m" | "10h" | "1d"
 <RequestExample>
 
 ```sh curl
-curl -XPOST https://qstash.upstash.io/v2/schedules \
+curl -XPOST https://qstash.upstash.io/v2/schedules/https://www.example.com/endpoint \
```
Comment (sancar) [inline qstash/api/schedules/create.mdx:67]: These examples also requires `Upstash-Cron` header.

File: _snippets/qstash-common-request.mdx [hunk 2]
```diff
@@ -66,3 +66,25 @@ strip efix and them to the destination API.
 
 example: "Upstash-Forward-My-Header: my-value" -> "My-Header: my-value"
 </ParamField>
+
+<ParamField header="Upstash-Callback-Forward-*" type="string" >
+ If you using the `Upstash-Callback` header to define a callback url,
```
Comment (sancar) [inline _snippets/qstash-common-request.mdx:71]: If you *are* using the ....

File: _snippets/qstash-common-request.mdx [hunk 3]
```diff
@@ -66,3 +66,25 @@ strip efix and them to the destination API.
...
+ you can specify the haeders sent along with the callback request using
```
Comment (sancar) [inline _snippets/qstash-common-request.mdx:72]: the haeders -> headers

File: _snippets/qstash-common-request.mdx [hunk 4]
```diff
@@ -66,3 +66,25 @@ strip efix and them to the destination API.
...
+
+To include a header in the callback request, prefix the header name with
+`Upstash-Callback-Forward-`. We will strip efix and them to the destination API.
+
+example: "Upstash-Callback-Forward-My-Header: my-value" -> "My-Header: my-value"
+</ParamField>
+
+<ParamField header="Upstash-Failure-Callback-Forward-*" type="string" >
+ If you using the `Upstash-Failure-Callback` header to define a callback url
+ when the delivery fails, you can specify the haeders sent along with the failure
```
Comment (mehmettokgoz) [inline _snippets/qstash-common-request.mdx:83]:
```suggestion
 when the delivery fails, you can specify the headers sent along with the failure
```

File: _snippets/qstash-common-request.mdx [hunk 5]
```diff
@@ -66,3 +66,25 @@ strip efix and them to the destination API.
...
+ callback request using the `Upstash-Failure-Callback-Forward-*` header.
+
+To include a header in the callback request, prefix the header name with
+`Upstash-Failure-Callback-Forward-`. We will strip efix and them to the destination API.
```
Comment (mehmettokgoz) [inline _snippets/qstash-common-request.mdx:87]:
```suggestion
To include a header in the callback request, add `Upstash-Failure-Callback-Forward-` prefix to the header name. We will strip this prefix and forward the header to the callback destination.
```

File: _snippets/qstash-common-request.mdx [hunk 6]
```diff
@@ -66,3 +66,25 @@ strip efix and them to the destination API.
...
+ the `Upstash-Callback-Forward-*` header.
+
+To include a header in the callback request, prefix the header name with
+`Upstash-Callback-Forward-`. We will strip efix and them to the destination API.
+
+example: "Upstash-Callback-Forward-My-Header: my-value" -> "My-Header: my-value"
+</ParamField>
+
+<ParamField header="Upstash-Failure-Callback-Forward-*" type="string" >
+ If you using the `Upstash-Failure-Callback` header to define a callback url
```
Comment (sancar) [inline _snippets/qstash-common-request.mdx:82]:
```suggestion
 If you are using the `Upstash-Failure-Callback` header to define a callback url
```

File: _snippets/qstash-common-request.mdx [hunk 7]
```diff
@@ -66,3 +66,25 @@ strip efix and them to the destination API.
...
 example: "Upstash-Forward-My-Header: my-value" -> "My-Header: my-value"
 </ParamField>
+
+<ParamField header="Upstash-Callback-Forward-*" type="string" >
+ If you are using the `Upstash-Callback` header to define a callback url,
+ you can specify the headers sent along with the callback request using
+ the `Upstash-Callback-Forward-*` header.
+
+To include a header in the callback request, prefix the header name with
+`Upstash-Callback-Forward-`. We will strip efix and them to the destination API.
```
Comment (sancar) [inline _snippets/qstash-common-request.mdx:76]: `Upstash-Callback-Forward-`. We will strip prefix and them to the destination API.

Comment (sancar) [inline _snippets/qstash-common-request.mdx:76]:
```suggestion
`Upstash-Callback-Forward-`. We will strip this prefix and forward the header to the callback destination.
```

**Review by ytkimirti — APPROVED** [review]

**Review by sancar — APPROVED** [review]

**Pull request merged**